### ISSUE TITLE
Generating writers: If case class has no tags, compilation fails with "tail of empty list"

### ISSUE THREAD

**Issue by rabejens** [fsanaulla/chronicler#239]
It is possible to have entries in InfluxDB that only consist of fields and timestamps. When trying to generate a writer for such a thing, compilation fails with :tail of empty list".

Steps to reproduce:
```scala
case class Foo(@timestamp time: Long, @tag channelid: String, @field value: Long)
implicit val writer = Influx.writer[Foo]
```

This works.

```scala
case class Foo(@timestamp time: Long, @field value: Long)
implicit val writer = Influx.writer[Foo]
```

yields:

```
java.lang.UnsupportedOperationException: tail of empty list
	at scala.collection.immutable.Nil$.tail(List.scala:471)
	at scala.collection.immutable.Nil$.tail(List.scala:466)
	at com.github.fsanaulla.chronicler.macros.InfluxImpl.unquote$1(InfluxImpl.scala:380)
	at com.github.fsanaulla.chronicler.macros.InfluxImpl.createWriteMethod(InfluxImpl.scala:433)
	at com.github.fsanaulla.chronicler.macros.InfluxImpl.writer(InfluxImpl.scala:460)

      implicit val writer = Influx.writer[Foo]
```

**Comment by fsanaulla** [comment]
Hey @rabejens 

Let me check it

**Comment by fsanaulla** [comment]
Fixed in #240
